backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/config/mobileui/MobileUIPickingUserProfile.java

```diff
@@ -52,6 +52,7 @@ public class MobileUIPickingUserProfile
 					.isAllowSkippingRejectedReason(false)
 					.createShipmentPolicy(CreateShipmentPolicy.DO_NOT_CREATE)
 					.isAllowCompletingPartialPickingJob(true)
+       			    .isShowLastPickedBestBeforeDateForLines(false)
```

Comment:
pls code format

Reply:
that's weird, this is how it looks in my IDE: 
![image](https://github.com/user-attachments/assets/f61f1e5e-8268-4ba5-80f1-24bf0d8288ec)